### PR TITLE
Update Templating landing page for V8

### PR DESCRIPTION
**Pull request by marcemarc**
This isn't showing the warning about check for V8 - but says that masterpages is still possible in V8 - which isn't so.

### CODE REVIEW

File: Reference/Templating/index.md [hunk 1]
```diff
@@ -1,44 +1,26 @@
...
-```
-
-#### Hybrid templates
-
-Umbraco can work with both MVC and Webforms templates at the same time, however there can only be one default. Umbraco will first check for an MVC view file before it checks for a Webforms master page file. For example, if a template was created in the backoffice called "Home", then when rendering a content page with this template Umbraco will check these locations and use the first one that is found:
+#### [Working with MVC (views, razor, etc...)](Mvc/index.md)
 
-~/Views/Home.cshtml
-~/Masterpages/Home.master
+Describes how to work with MVC views, the razor syntax and APIs available as well details on how to create forms, step-by-step guides and other advanced techniques.
```
Comment (jmayntzhusen) [inline Reference/Templating/index.md]:
```suggestion
Describes how to work with MVC views, the razor syntax and APIs available. It also describes how to create forms, has some step-by-step guides and other advanced techniques.
```
Found this sentence a bit confusing. Tried to make it less so, don't think it's great yet though 😁

Comment (marcemarc) [reply Reference/Templating/index.md]: yes I didn't write that :-P

File: Reference/Templating/index.md [hunk 2]
```diff
@@ -1,44 +1,26 @@
...
-_Templating in Umbraco consists of 2 larger concepts, namely Templates, and Macros. Templates are used for the HTML layout of your pages, whereas macros are reusable dynamic components used for embedding navigation, forms, lists, and so-on in your templates._
+_Templating in Umbraco consists of 3 larger concepts, namely Templates (Views), Partials (Partial Views) and Macros (Macro Partials). Templates are used for the HTML layout of your pages, partials can be included in your templates for shared functionality across different page templates and macros for reusable dynamic components that can be controlled by editors to embed functionality into the grid or rich text areas._
 
-## Templates
+## Templating technology
 
-There are 2 types of templating technologies in Umbraco:
+In Umbraco V8 there is now only one type of templating technology:
 
 * MVC (views)
```
[the remark this answers is not in the document]
Comment (marcemarc) [reply Reference/Templating/index.md]: Updated!

**Comment by jmayntzhusen** [comment]
Awesome, will merge 🙂

**Pull request merged**